### PR TITLE
Update calendar-ui package, add localization settinsg if calendar plugin isn't installed

### PR DESCRIPTION
**Pull request by liamcain**
My very basic/dumb approach to localization here is: 

> if the calendar plugin is installed, use those settings. If not, show localization settings within slated.

It's far from optimal but it is at least functional. Open to ideas for a better approach!

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -4067,8 +4067,10 @@ rollup@2.35.1:
   optionalDependencies:
     fsevents "~2.1.2"
 
-"rrule@file:../rrule":
+"rrule@\rrule":
```
Comment (liamcain) [inline yarn.lock:4070]: hmm, this was pointing at a local file previously. It was compiling on my machine. Let me revert this change, but I think a version should be pinned instead? Or a git hash if you have a fork?

Comment (liamcain) [reply yarn.lock]: Linting is failing because it can't install rrule

**Comment by tgrosinger** [comment]
Ah shoot, yeah the rrule package is a mess right now. I am using a branch from a fork because it has a required bug fix that the maintainer has not yet merged in. Let me see if I can have npm use that forks commit hash instead of the NPM published version.

File: src/main.ts [hunk 2]
```diff
@@ -1,3 +1,8 @@
+/* eslint-disable */
```
Comment (tgrosinger) [inline src/main.ts:1]: Why's this necessary here?

File: yarn.lock (hunk 1 above)
Comment (tgrosinger) [reply yarn.lock]: Here's the fix I am imaging, however it's not quite working yet. Builds seem to work fine, but tests complain that it can't find rrule. I'll take another look this evening.

https://github.com/tgrosinger/slated-obsidian/pull/35

```
$ ./yarn test 
yarn run v1.22.5
$ jest
 FAIL  tests/task-line.test.ts
  ● Test suite failed to run

    src/task-line.ts:15:34 - error TS2307: Cannot find module 'rrule' or its corresponding type declarations.

    15 import RRule, { Frequency } from 'rrule';
```

File: src/main.ts (hunk 2 above)
Comment (liamcain) [reply src/main.ts:1]: Eslint will try turning this into:
```
import type moment, { WeekSpec } from 'moment';
```

which isn't valid Typescript.

But let me fix this since there's a better approach outlined here: https://github.com/benmosher/eslint-plugin-import/issues/1869

File: yarn.lock (hunk 1 above)
Comment (tgrosinger) [reply yarn.lock]: It should be fixed now.

**Comment by tgrosinger** [comment]
If you rebase this on the main branch it should become mergable. I don't seem to be able to do that myself.

**Comment by tgrosinger** [comment]
Thank you!
